src/ui/FormElement.tsx

```diff
@@ -112,6 +120,15 @@ const FormElement = ({
         width={labelPosition === LabelPositions.RIGHT ? 'auto' : '100%'}
         minWidth={50}
       >
+        {maxLength && (
```

Comment:
```suggestion
        {typeof maxLength !== 'undefined' && (
```

maxLength 0 is a bit stupid, but better to be explicit to avoid bugs where 0 evaluates to false